### PR TITLE
Re-Add slack plugin

### PR DESCRIPTION
**Pull request by maddinador**
Made a mistake when switching rthill's fork from upstream:master to upstream:develop. @rthill: sorry for messing things up!
Thus I started a clean fork and re-added the slack dir and its three files.
Original discussion was at pull request #144 and shall be continued here.

### CODE REVIEW

File: slack/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+SlackInstance_1:
+    plugin_name: slack
+    instance: WorkspaceYourTeam
+    token: abc/def/ghi # Token for posting to workspace '<your_team>'
+	
+SlackInstance_2:
+    plugin_name: slack
+    instance: WorkspaceAnotherTeam
+    token: jkl/mno/pqr # Token for posting to another workspace '<another_team>'
+</pre>
```
Comment (ohinckel) [inline slack/README.md]: Can you change the example to use
```
```yaml
....
```

... instead of these `<pre>` tags?

```

File: slack/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# smarthome-slack
+Plugin to send push notifications to Slack 
+
+## Installation
+<pre>
+cd smarthome.py directory
+cd plugins
+git clone https://github.com/rthill/slack.git
+</pre>
```
Comment (ohinckel) [inline slack/README.md]: Do we need some installation instructions? Usually the plugins repos is checked out completely in a given version.

But if we need, can you change the repo URL to point to the SHNG repository?

**Comment by maddinador** [comment]
@ohinckel thanks for your feedback! 👍 
I wasn't aware of the nice ```-Markdown-Feature with syntax highlighting for different languages! Very great!
Ayn other feedback? 😁

**Comment by ohinckel** [comment]
Not so far from my side. Should be ready to be merged.

Do you have some lines of text about this plugin which we can use in our release notes?

**Comment by maddinador** [comment]
just a quick proposal:

**en**
Slack:
- introduced new notification plugin
- Slack stands for „Searchable Log of All Conversation and Knowledge“
- allows you to send instant messages with programmable text to your Slack workgroups
- helps you to keep track of predefined events

**de**
Slack:
- neues Benachrichtigungs-Plugin hinzugefügt
- Slack steht für „Searchable Log of All Conversation and Knowledge“ (dt. durchsuchbares Protokoll aller Konversationen und des Wissens)
- Sofort-Nachrichten mit programmierbarem Text an die Slack Arbeitsgruppen senden
- Unterstützt bei der Überwachung definierter Events

**Comment by ohinckel** [comment]
Added to [release notes](https://github.com/smarthomeNG/smarthome/wiki/SmartHomeNG-v1.6---geplante-Features/_compare/8a648e806edc89cf65fb555883bc43491972a608...f7537669ceeba8e3e90fac93033f248eaac7ecec).

**Pull request merged**